.github/workflows/sync.yaml

```
@@ -83,4 +81,58 @@ jobs:
+        env:
+          from: ${{ inputs.from }}
+          to: ${{ inputs.to }}
+          image: eeems/atomic-arch
+
+  sync:
+    name: Sync eeems/atomic-arch:${{ matrix.tag }}
+    runs-on: ubuntu-latest
+    if: ${{ needs.matrix.outputs.items > 0 }}
+    needs: matrix
+    strategy:
+      matrix: ${{ fromJson(needs.matrix.outputs.tag-matrix) }}
```

actionlint (1.7.8)

property "tag-matrix" is not defined in object type {items: string; variant-matrix: string}

(expression)